src/executables/uber-time.js

```diff
@@ -8,13 +8,19 @@ import CommandExecutionService from '../services/CommandExecutionService';
 
 const service = new CommandExecutionService();
 
+const address = program.args[0].trim().toString();
 program
   .arguments('<address>')
   .action((address) => {
+    if (!address.length) {
```

Comment:
Oh, yes, I see what you're saying here. Will remove. Let's do a squash merge when this is done!